README.md

```diff
@@ -229,19 +229,25 @@ certain special constraints, like if the mutex is never contested (you might be
 re-using multi-threaded code in a single-threaded context), or if it's only
 contested at times when a user is expecting a glitch anyway (if, say, an audio
 device is disconnected). RADSan always assumes the worst, and this may not be
-true for your particular use case. You can turn off RADSan's error detection
-temporarily by wrapping code in `radsan_off()` and `radsan_on()` as follows:
+true for your particular use case. 
+
+You can turn off RADSan's error detection for such a call by extracting the 
+behavior to a function, and adding adding the `no_sanitize` attribute:
+
 
 ```cpp
-#include "radsan.h" // (found in llvm-project/compiler-rt/lib/radsan)
+__attribute__((no_sanitize("realtime")))
+void mutex_unlock_uncontested (std::mutex& m)
```

Comment:
I think "contended" is more common than "contested" in the context of a mutex, at least according to Google, which has about 5x more hits for "contended"! It's obviously not a big deal, but while you're fixing the conflicts with the latest README, would you be up for a quick find/replace to switch the many "contested"s that are in this file (and there are a bunch from before this change!) to "contended"? 

Otherwise this looks all good to go from my perspective!

Reply:
Done! 

I ran into the conflict with your radsan_on/radsan_off change. Let me know if you would like that back in addition to this change. I opted to remove

Reply:
Cool, thanks! Happy for that to be removed - good call, it was redundant now these changes are going in.